### PR TITLE
A lot of template fixes

### PR DESCRIPTION
**Pull request by emanuelschuetze**
- Reset password from current (unsaved) value of default_password.
- MotionForm, AssignmentForm and UserForm factory for open ngDialog.
  Fixes edit url redirect problem.
- Added back-to-detail-view-button to list of speakers view.
- Add translation context.

### CODE REVIEW

**Comment by ostcar** [comment]
ok, please rebase and merge

**Comment by emanuelschuetze** [comment]
Rebased. 
@normanjaeckel Please review at least the *.py files. Then please merge.

File: openslides/users/views.py [hunk 1]
```diff
@@ -153,10 +153,11 @@ def destroy(self, request, *args, **kwargs):
     @detail_route(methods=['post'])
     def reset_password(self, request, pk=None):
         """
-        View to reset the password (using the default password).
+        View to reset the password using the requested password.
         """
         user = self.get_object()
-        user.set_password(user.default_password)
+        user.set_password(request.data['password'])
```
Comment (normanjaeckel) [inline openslides/users/views.py]: Can we set the new password to the database default password if the client sends no new password?

File: openslides/agenda/static/js/agenda/site.js [hunk 2]
```diff
@@ -198,6 +199,13 @@ angular.module('OpenSlidesApp.agenda.site', ['OpenSlidesApp.agenda'])
             $scope.speakers = $filter('orderBy')(item.speakers, 'weight');
         });
 
+
+        // go to detail view of related item (content object)
+        $scope.open = function (item) {
+            $state.go(item.content_object.collection.replace('/','.')+'.detail',
+                    {id: item.content_object.id});
```
Comment (normanjaeckel) [inline openslides/agenda/static/js/agenda/site.js]: Indention is broken.

File: openslides/users/views.py (hunk 1 above)
Comment (normanjaeckel) [inline openslides/users/views.py]:
```python
if request.data.get('password'):
    user.default_password = request.data['password']
user.set_password(user.default_password)
user.save()
```

File: tests/integration/users/test_viewset.py [hunk 3]
```diff
@@ -165,9 +165,12 @@ def test_reset(self):
...
         user.default_password = 'new_password_Yuuh8OoQueePahngohy3'
         user.save()
-        response = admin_client.post(reverse('user-reset-password', args=[user.pk]))
+        response = admin_client.post(
+            reverse('user-reset-password', args=[user.pk]),
+            {'password': 'new_password_Yuuh8OoQueePahngohy3'})
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertTrue(User.objects.get(pk=user.pk).check_password('new_password_Yuuh8OoQueePahngohy3'))
+        self.assertTrue(User.objects.get(pk=user.pk).check_password(
+            'new_password_Yuuh8OoQueePahngohy3'))
```
Comment (normanjaeckel) [inline tests/integration/users/test_viewset.py]: Proposal: Add a new test that the password is reset even if no new password is send.

**Comment by normanjaeckel** [comment]
Some remarks. Also travis failed.

**Comment by emanuelschuetze** [comment]
All done. Ready for review.

**Comment by normanjaeckel** [comment]
Rebase and merge.

**Pull request merged**